docs/docsite/rst/network_aggregate_resources.rst

```diff
@@ -0,0 +1,175 @@
+*********************************
+Aggregate Resources in Networking
+*********************************
+
+.. contents:: Topics
+
+This section explores how and when `Aggregate Resources` can be used within Ansible Networking.
+
+This document is part of a collection on Networking. The complete list of guides can be found at :ref:`Network Guides <guide_networking>`.
+
+Overview
+========
+
+Consider the case if you wanted to ensure a set of vlans are present on a switch. Although at a high level this sounds like a simple request, there in a slight nuance:
+
+* Should these vlans be in *addition* to the existing configuration?  - *additive*
```

Comment:
Please spell these out. Something like: "If the vlans are to be in addition to the existing configuration, they are considered *additive* vlans".